### PR TITLE
Modified and re-exported the logo.

### PR DESCRIPTION
**Pull request by Samuwhale**
Also included the affinity designer file this time.

### CODE REVIEW

**Comment by qubixes** [comment]
@Samuwhale Thanks for the PR, looks great! I would like to merge this PR after we have a new release. The current plan is:
- Last release for MetaSynth, including the regex update.
- Then make updates to rename MetaSynth -> metasyn
- Create a new release/package on PyPi with the new name.

**Review by vankesteren — APPROVED** [review]
Yes, perfect. As @qubixes said, let's wait a little bit before merging.

**Pull request merged**
